tutorials/tour/_posts/2017-02-13-classes.md

```diff
@@ -9,46 +9,103 @@ categories: tour
 num: 4
 next-page: traits
 previous-page: unified-types
+topics: classes
+assumed-knowledge: no-return-keyword, type-declaration-syntax, string-interpolation, procedures
 ---
 
-Classes in Scala are static templates that can be instantiated into many objects at runtime.
-Here is a class definition which defines a class `Point`:
+Classes in Scala are blueprints for creating objects. They can contain methods,
+values, and variables, which are collectively called _members_.
+
+## Defining a class
+A minimal class definition is simply the keyword `class` and
+an identifier:
+```tut
+class User
+
+val user1 = new User
+```
+The keyword `new` is used to create an instance of the class. `User` has an implicit constructor which takes no arguments because no constructor was defined. However, you'll often want a constructor and class body. Here is an example class definition for a point:
```

Comment:
I would say "default" rather than "implicit" here to avoid confusion with `implicit` which is an unrelated concept.